### PR TITLE
Serg

### PR DESCRIPTION
**Pull request by poliakosOSU**
Made some changes to Board. Constructor and getShips() should be good.

### CODE REVIEW

File: src/main/java/cs361/battleships/models/Board.java [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 	//*******ERRORS with my array, because arrays start at 0 and not at 1
 	Square[][] grid = new Square[10][10];
+	ArrayList<Ships> currentShips = new ArrayList<Ships>();
```
Comment (MannyB6) [inline src/main/java/cs361/battleships/models/Board.java]: May want to see if we are looking to use a list or an arrayList, I know we can have `List<Result> test = new ArrayList<result>` and then just add from there

File: src/main/java/cs361/battleships/models/Board.java [hunk 2]
```diff
@@ -3,38 +3,98 @@
...
+public class Board {
 
 	/*
 	DO NOT change the signature of this method. It is used by the grading scripts.
 	 */
+
+	private Square[][] grid;
+
+	private List<Ship> shipList;
+	private List<Result> attackList;
```
Comment (MannyB6) [inline src/main/java/cs361/battleships/models/Board.java]: I believe we should declare shipList and attackList outside.

Comment (poliakosOSU) [reply src/main/java/cs361/battleships/models/Board.java]: You're wrong

**Comment by PembleC** [comment]
This is looking really good guys. Team Works

File: src/main/java/cs361/battleships/models/Board.java (hunk 2 above)
Comment (PembleC) [reply src/main/java/cs361/battleships/models/Board.java]: It is fixed MANNY!!

**Pull request closed without merging**